oneview_redfish_toolkit/tests/api/test_zone.py

```diff
@@ -77,6 +79,7 @@ def test_drives_as_links_when_storage_controllers_are_not_configured(self):
         zone = Zone(zone_id,
                     profile_template,
                     self.server_hardware_list,
+                    self.enclosure_name_none,
```

Comment:
Why dont pass simply `None`? Instead of declare a new variable to assign `None`.